tiledb/sm/query/writers/writer_base.h

```diff
@@ -398,20 +413,27 @@ class WriterBase : public StrategyBase, public IQueryStrategy {
   Status split_coords_buffer();
 
   /**
-   * Writes all the input tiles to storage.
+   * Writes a number of the input tiles to storage for all
+   * dimensions/attributes.
    *
+   * @param start_tile_idx Index of the first tile to write.
+   * @param end_tile_idx Index of the last tile to write.
+   * @param frag_meta Current fragment metadata.
    * @param tiles Attribute/Coordinate tiles to be written, one element per
    *     attribute or dimension.
-   * @param tiles Attribute/Coordinate tiles to be written.
    * @return Status
    */
   Status write_all_tiles(
```

Comment:
Nit: Should this be renamed to just `write_tiles` now? It doesn't write all of the tiles, just tiles from `start_tile_idx` to `end_tile_idx`.